### PR TITLE
Cleared up imports and comment exported function

### PR DESCRIPTION
**Pull request by Heasummn**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -5,8 +5,8 @@ import (
 	"os/signal"
 	"syscall"
 
-	"./config"
-	"./safetyjim"
+	"github.com/Samoxive/SafetyJim/config"
```
Comment (Samoxive) [inline main.go]: Why use github imports here instead of relative path?

**Comment by Heasummn** [comment]
The import syntax has been fixed. Go standard dictates putting your projects in `$GOPATH/src`, the new code should still work if that has not been done, however the current code will not work if one follows Go standard. Go recommends, and sometimes requires absolute imports, so we're making use of them.

**Comment by Samoxive** [comment]
This version still rises problems for me, does the current version not work when put into %gohome%/src?

**Comment by Samoxive** [comment]
Thanks for the PR, now we can import subpackages the right way :P

**Pull request merged**